inscriptions/ISic001266.xml

```diff
@@ -190,32 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140755">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    
-                    <expan>
-                        <abbr>Θ</abbr>
-                        <ex>εοῖς</ex>
-                    </expan>
-                    <expan>
-                        <abbr>Κ</abbr>
-                        <ex>α</ex>
-                        <abbr>τ</abbr>
-                    </expan>
-                    (αχθονίοις).<lb n="2"/>Χαίρο<supplied reason="omitted">υ</supplied>σα αἰ-<lb n="3"/>εὶ γλυκυτάτη<lb n="4"/>ζήσασα ἔτη
-                    <supplied reason="lost">—</supplied>
-                    <lb n="5"/>πατὴρ τ
-                    <supplied reason="lost">ῇ ἰδίᾳ</supplied>
-                    <lb n="6"/>
-                    <supplied reason="lost">θυ</supplied>γατρ<supplied reason="lost">ὶ ἐποίησεν</supplied>.
-                    <lb n="7"/>
+                    <lb n="1"/><w><expan><abbr>Θ</abbr><ex>εοῖς</ex></expan></w> <w><expan><abbr>Κ</abbr><ex>α</ex><abbr>τ</abbr><ex>αχθονίοις</ex></expan></w>
+                    <lb n="2"/><w>Χαίρο<choice><corr>υ</corr><sic>ι</sic></choice>σα</w> <w>αἰ
+                    <lb n="3" break="no"/>εὶ</w> <w>γλυκυτάτη</w>
+                    <lb n="4"/><w>ζήσασα</w> <w>ἔτη</w> <gap reason="lost" extent="unknown" unit="character"><desc>numeral</desc></gap>
+                    <lb n="5"/><w>πατὴρ</w> <w>τ<supplied reason="lost">ῇ</supplied></w> <w><supplied reason="lost">ἰδίᾳ</supplied></w>
```

Comment:
this may be personal (since both are permitted by the TEI, but I woudl put the w inside the supplied, rather than the other way around, when it is the whole word that is supplied.